### ISSUE TITLE
please

### ISSUE THREAD

**Issue by M4gics** [extremecoders-re/go-dispatch-proxy#12]
I am exploring the implementation of a SOcks5 server that, once the server receives a request, takes turns using multiple Socks5 proxy access targets in the configuration file，I hope I can get your guidance

**Comment by extremecoders-re** [comment]
This is already implemented in gdp. Check the [Load balance SSH tunnels](https://github.com/extremecoders-re/go-dispatch-proxy#2---load-balance-ssh-tunnels) feature.

However this needs to have the tunnels previously set up. 
In case you also want to setup the tunnels from code, you can use any decent SSH client library for Go.

**Comment by SirSAC** [comment]
Hi M4gics, u want the program to have a proxy chain?
Like u set two interfaces and then set some proxyes like as an example Privoxy and Squid to be used by go-dispatch-proxy, right?

**Comment by M4gics** [comment]
Yes, can Load Balancing be more concise and implemented on the SOcks5 server ? I need my tool can also use proxy chain, and I hope I can learn from relevant demo code




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Sergiu Alexandru ***@***.***&gt;; 
发送时间: 2021年9月13日(星期一) 上午10:55
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [extremecoders-re/go-dispatch-proxy] please (#12)






你好 M4 gics ，你想让程序有一个代理链？
比如 u 设置两个接口，然后设置一些 proxyes ，比如 Privoxy 和 Squid ，让 go - dispatch - proxy 使用，对吧？

—
您正在接收此消息，因为您创建了该线程。
直接回复此邮件，在 GitHub 上查看或取消订阅.
GitHub Mobile 上的分流通知监督办或机器人.

**Comment by SirSAC** [comment]
yes it is on SOCKS5 but at this moment my program and extremecoders can't intercept multiple proxy, its hard to implement this thing and GoLang in my opinion is good but not mature code yet, so i not have any ideea how to implement a proxy chain future without broke other platforms and at this moment i try to make my program to work on Android. Thanks for reply and im sry but i can't help more about this issue.

**Comment by M4gics** [comment]
ok I&nbsp; try to find other materials to achieve




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Sergiu Alexandru ***@***.***&gt;; 
发送时间: 2021年9月13日(星期一) 下午5:49
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [extremecoders-re/go-dispatch-proxy] please (#12)






是的，就在 SOCKS 5 上但目前我的程序和 extreme coders 不能拦截多个代理，它很难实现这个东西和 Go Lang 在我看来是好的，但还不成熟的代码，所以我不知道如何实现代理链的未来没有打破其他平台，在这一刻，我试图使我的程序工作在 Android 上。感谢回复而 imsry但我不能帮助更多关于这个问题。

—
您正在接收此消息，因为您创建了该线程。
直接回复此邮件，在 GitHub 上查看或取消订阅.
GitHub Mobile 上的分流通知监督办或机器人.

**Comment by SirSAC** [comment]
At this moment i make the program to work on Android but its useless because carrier network form SIM card is still changing his IP address, so at this moment i need to make an auto IP obtain but i want to implement Cloudflare WARP into it for really insane speed because WARP is doing more than a DNS query, its compressing the sites and links data on Cloudflare servers before to received the site or link data, if u can help me with this i will really appreciate and suggest u as a good programmer.
